cmd/operator/deploy/operator/11-rule-evaluator.yaml

```diff
@@ -103,8 +103,8 @@ spec:
           limits:
             memory: 1G
           requests:
-            cpu: 8m
-            memory: 32M
+            cpu: 1m
+            memory: 16M
```

Comment:
Should we maybe keep this consistent with `cmd/operator/deploy/rule-evaluator/01-deployment.yaml`?

Reply:
Thx - done!